src/components/Delegate/StakingOpportunities.tsx

```diff
@@ -198,8 +194,7 @@ export const StakingOpportunities = ({ assetId }: StakingOpportunitiesProps) =>
     [accountSpecifier],
   )
 
-  // Special case for osmosis because we do yet support staking
```

Comment:
This is fine because the staking opportunities dont show up if osmosis feature flag is off